heat/core/manipulations.py

```diff
@@ -895,23 +890,16 @@ def unique(a, sorted=False, return_inverse=False, axis=None):
         a.comm.Bcast(indices, root=max_pos)
         gres = local_data[indices.tolist()]
 
-        is_split = a.split
+        # is_split = a.split
```

Comment:
remove dead code / is this ever set again? I see it used in the array call later